src/renderer/screens/loader/LoaderScreen.scss

```diff
@@ -57,7 +57,7 @@
 
   .LoaderProgress {
     position: absolute;
-    margin-top: 7em;
+    margin-top: 15em;
```

Comment:
I _think_ this is to support Corda, I'll look into this in `revert/corda` branch